### ISSUE TITLE
Add "Message keys" to rule documentations

### ISSUE THREAD

**Issue by henriquemoody** [Respect/Validation#1089]
**TL; DR**

Pick one rule and update its documentation with a session like the _"Message keys"_ session below. 

***
Many rules have _message keys_ that are strings surrounded by `{{` and
`}}` that are replaced after a validation fails. A lot of these keys are
documented but not all of them.

This issue is so we can have an specific session in the documentation of
rule that has suck message keys. These sessions can look like the text
bellow using Equals as an example.

***
### Message keys

Key             | Description
----------------|-------------
`{{name}}`      | Input or user-defined name of the rule
`{{compareTo}}` | Value used to compare with the input

**Comment by henriquemoody** [comment]
There are more things I would like to do with the messages, and I'm closing old issues.
